app/code/Magento/Backend/Controller/Adminhtml/Auth/Login.php

```diff
@@ -69,4 +97,26 @@ private function getRedirect($path)
         $resultRedirect->setPath($path);
         return $resultRedirect;
     }
+
+    /**
+     * Verify if correct backend uri requested.
+     *
+     * @return bool
+     */
+    private function isValidBackendUri(): bool
```

Comment:
@engcom-Foxtrot 
TBH it's better to cover this functionality with some unit or integration test. What do you think?